MetaMask/metamask-mobile
# fix: cherry pick PR improvement

## **Description**

This PR fixes fetching of all the branches needed to create the cherry pick PRs. 

Linked test PR that was created from action: 

## **Related issues**

Fixes: NA

## **Manual testing steps**

1. Go to the action page & select "Create Cherry Pick PR"
2. Select run workflow from `fix/cherry-pick-node-version`
3. Enter base branch, commit hash and PR number
4. PR should be successfully created unless conflicts occur in the creation and action should fail. 

## **Screenshots/Recordings**

Link successful PR runs

### **Before**

https://github.com/MetaMask/metamask-mobile/actions/runs/6867014033

### **After**

Successful cherry-pick: https://github.com/MetaMask/metamask-mobile/pull/8316
https://github.com/MetaMask/metamask-mobile/actions/runs/7562792265

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
